scripts/deploy.js

```diff
@@ -0,0 +1,93 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+const { ethers } = require("hardhat");
+const fs = require('fs')
+const { NFTStorage, File } = require('nft.storage')
+const dotenv = require('dotenv');
+const { MerkleTree } = require('merkletreejs')
+const keccak256 = require('keccak256')
+dotenv.config();
+
+const whitelistAddresses = [
+    '0xB4599439114a6a814218254008ed5c60D0d8049d'
+]
+
+
+async function uploadData() {
+    const client = new NFTStorage({ token: process.env.NFT_STORAGE_API_KEY });
+
+    const imageData = fs.readFileSync(`./assets/og_token.jpg`)
+    const imageFile = new File([imageData], `W3B_OG_TOKEN.jpg`, { type: 'image/jpg' });
+    const imageCID = await client.storeBlob(imageFile);
+    console.log('image cid', imageCID);
+
+    const animationData = fs.readFileSync(`./assets/og_token.mp4`)
+    const animationFile = new File([animationData], `W3B_OG_TOKEN.mp4`, { type: ' video/mp4' });
+    const animationCID = await client.storeBlob(animationFile);
+    console.log('animation cid', animationCID);
+
+
+    return {
+        imageCID,
+        animationCID
+    }
+
+}
+
+
+
+async function main() {
+    // let's first create some random address
+    const whitelist = Array.from(Array(10).keys()).map(_ => ethers.Wallet.createRandom().address)
+    // add my own account for testing:
+    const account = await ethers.getSigner()
+    const myAddress = account.address
+    console.log(myAddress);
+    whitelist.push(myAddress)
+
+    const leafNodes = whitelist.map(addr => keccak256(addr))
+    const merkleTree = new MerkleTree(leafNodes, keccak256, { sortPairs: true })
+    const merkleRoot = merkleTree.getRoot()
+
+    console.log('merkle root', merkleRoot);
+
+    const { imageCID, animationCID } = await uploadData()
+
+    const Contract = await ethers.getContractFactory('MunichWeb3Builders');
```

Comment:
Maybe let's rename to `MunichWeb3BuildersContract`?
I think the current name can lead to confusion with `ethers.Contract`: https://docs.ethers.io/v5/single-page/#/v5/api/contract/contract/

Reply:
Yes agree, thanks!